java-symbolic-execution/src/main/java/org/sonar/java/se/ExplodedGraphWalker.java

```diff
@@ -734,6 +737,20 @@ private void visit(Tree tree, @Nullable Tree terminator) {
       case SWITCH_EXPRESSION:
         programState = programState.stackValue(constraintManager.createSymbolicValue(tree));
         break;
+      case DEFAULT_PATTERN:
+        programState = programState.stackValue(constraintManager.createSymbolicValue(tree));
+        break;
+      case TYPE_PATTERN:
+        programState = programState.stackValue(constraintManager.createSymbolicValue(tree));
+        break;
+      case NULL_PATTERN:
+        // FIXME might cause NPE in the block
```

Comment:
From this message, it is not clear if the analyzer is subject to a NPE or if the code we are analyzing could be subject to a NPE.
I guess it's the latter, so we could go for something like:
```suggestion
        // FIXME we could add a null constraint to "o"
```